contrib/metacontroller/UPDGRADE.md

```diff
@@ -0,0 +1,11 @@
+# Upgrade Metacontroller
+
+Metacontroller is pulled from [Kubeflow Pipelines third-party folder](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/third-party/metacontroller). To update this component specify the desired KFP version and run the following command in console from the root directory **/**:
+
+```bash
```

Comment:
NIT: Maybe this command can be a `Makefile rule` as well?

Reply:
Added. Thank you!